src/main/java/edu/tamu/scholars/middleware/config/WebSecurityConfig.java

```diff
@@ -218,6 +219,10 @@ protected SecurityFilterChain configure(HttpSecurity http) throws Exception {
             .authorizeHttpRequests(authz -> authz
                 .requestMatchers("/login/saml2/**", "/saml2/**")
                     .permitAll()
+
+                .requestMatchers(POST, "/individual/export").permitAll()
+
+                .requestMatchers(POST, "/individual/*/export").permitAll()
```

Comment:
```suggestion
                .requestMatchers(POST, "/individual/{id}/export").permitAll()
```
However, they can be removed.

https://github.com/TAMULib/scholars-discovery/blob/tamu-main/src/main/java/edu/tamu/scholars/middleware/export/controller/IndividualExportController.java#L59C17-L59C101

Admin guard clause in the controller.